packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -44,6 +44,33 @@ export class ListboxOption extends FoundationElement {
      */
     public proxy: HTMLOptionElement;
 
+    /**
+     * The checked state is used when the parent listbox is in multiple selection mode.
+     * To avoid accessibility conflicts, the checked state should not be present in
+     * single selection mode.
+     *
+     * @public
+     */
+    @observable
+    public checked?: boolean;
```

Comment:
Should this be an attribute as well?

Reply:
It's not intended to be configurable as an attribute since the rules regarding focus wouldn't allow it:

> When a multi-select listbox receives focus:
> * If none of the options are selected before the listbox receives focus, focus is set on the first option and there is no automatic change in the selection state.
> * If one or more options are selected before the listbox receives focus, focus is set on the first option in the list that is selected.

In our case, "focus" will be handled by this checked state (specifically the `aria-checked` attribute).